apps/admin/src/hooks/user-preferences.ts

```diff
@@ -30,7 +47,7 @@ export const useUserPreferences = (): UseQueryResult<Preferences> => {
             const raw = user.accessibility || "{}";
             const parsed = JSON.parse(raw) as unknown;
 
-            return PreferencesSchema.parse(parsed);
+            return PreferencesCodec.parse(parsed);
```

Comment:
I don't think we should rename the schema. The codec thing is an implementation level concern of making the transforms bidirectional. It's still a schema.